src/golive/README.md

```diff
@@ -0,0 +1,8 @@
+# Going Live
+
+You successfully deployed your project to Platform.sh? Time to go live. This is a simple process, but it's worth reading the docs the first time around.
+
+* [Pre-Launch Checklist](/golive/checklist.md) 
+* [Going Live Steps](/golive/steps.md) 
+* [Troubleshooting](/golive/troubleshoot.md) 
+* [Example](/golive/example.md)   
```

Comment:
This file is never seen or linked from.  It also doesn't really offer anything, so I will just remove it.